client/settings/express-checkout-settings/platform-checkout-settings.js

```diff
@@ -4,20 +4,32 @@
  */
 import React from 'react';
 import { __ } from '@wordpress/i18n';
-import { Card, CheckboxControl, TextControl } from '@wordpress/components';
+import {
+	Card,
+	CheckboxControl,
+	TextControl,
+	BaseControl,
+} from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import CardBody from '../card-body';
+import BrandingFileUpload from '../../card-readers/settings/file-upload';
```

Comment:
This component doesn't look like a generic component. Should we have our own component extending the generic `file-upload` component?

Reply:
Currently WCPay Server sends the Stripe account branding logo to WooPay, that's the same logo that we set on Card Reader receipts, so I thought there was no problem to use the same component.

Reply:
Oh I didn't know about this Figma file, I'll change the PR to follow this design.